File: FurnitureAPI/FurnitureAPI/Models/UserModel.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System;

namespace FurnitureAPI.Models
{
    public class UserModel
    {
        
        public int Id { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime? BirthDate { get; set; }
        public string Role { get; set; }
    }
}

File: FurnitureAPP/app/src/main/java/com/example/furnitureapp/datalayers/retrofit/models/UserModel.kt
package com.example.furnitureapp.datalayers.retrofit.models

import com.google.gson.annotations.SerializedName

data class UserModel(
    @SerializedName("id")
    val id: Int,
    @SerializedName("email")
    val email: String,
    @SerializedName("firstName")
    val firstName: String,
    @SerializedName("lastName")
    val lastName: String,
    @SerializedName("birthDate")
    val birthDate: String,
    @SerializedName("role")
    val role: String
)
